### DISCUSSION TITLE
Clearing a form after save (DONE)

### DISCUSSION THREAD

**Discussion by mjashanks** [Budibase/budibase#2059] [Ideas]
It would be useful to have the ability to have a "Clear Form" action, for button click.

The use case is described here: https://github.com/Budibase/budibase/discussions/2043

**Comment by KernOil** [comment]
YES PLEASE, I WOULD USE THIS!!

**Comment by gvlx** [comment]
Close this issue.

Version 0.9.185 already has this feature:  
![imagem](https://user-images.githubusercontent.com/67311/141470131-f0f0ced2-8939-4b45-a931-98cde205804b.png)


Nice work :)
